Теперь можно залезать на стойки(RACK)

Breyn78 opened the pull request:
Fixes #180 
Теперь вы можете залезать на стойки(Rack)
🆑 Breyn578
- tweak: На стойки теперь можно залезть как на столы и ящики.
<!--
Подробно про оформление ПРов можно прочитать тут: https://github.com/TauCetiStation/TauCetiClassic/wiki/Styling-of-Pull-Requests-for-Dummies
Там же написано про то, как сделать чейнджлог. 

!!!
Если авторство не полностью ваше, вы делаете порт с другого билда - укажите первоисточник изменений!
Для крупных комплексных изменений достаточно будет указать билд(ы)-первоисточник, в остальных случаях можете указать исходный ПР.
!!!

Для копипаста:
Список классификаторов для быстрого копирования: bugfix, rscadd, rscdel, image, sound, spellcheck, tweak, balance, map, performance, experiment.

Пример списка:
:cl:
 - image: Добавлен плакат с изображением статного мужчины с конусом на голове и арбузами вокруг него.
 - image: С плаката чужого в форме горничной убрана цензура.
-->

Breyn78 committed:
Merge pull request #1 from GammaStation/master

Последние обновления

Breyn78 committed:
Теперь можно залезть на RACK

Теперь неприступные стойки остались позади, вы можете залезать на них когда хотите!

LudwigVonChesterfield commented on code/game/objects/structures/tables_racks.dm:
```diff
@@ -727,6 +727,9 @@
 	var/parts = /obj/item/weapon/rack_parts
+	climbable = 1
+//	mouse_drag_pointer = MOUSE_ACTIVE_POINTER	//???
+	var/rigged = 0
```
А это зачем?

LudwigVonChesterfield commented on code/game/objects/structures/tables_racks.dm:
```diff
@@ -727,6 +727,9 @@
 	throwpass = 1	//You can throw objects over this, despite it's density.
 	var/parts = /obj/item/weapon/rack_parts
+	climbable = 1
+//	mouse_drag_pointer = MOUSE_ACTIVE_POINTER	//???
```
А комментированный код зачем красть?

Breyn78 committed:
Update tables_racks.dm

Miracler112 commented on code/game/objects/structures/tables_racks.dm:
```diff
@@ -728,7 +728,6 @@
 	var/parts = /obj/item/weapon/rack_parts
 	climbable = 1
-//	mouse_drag_pointer = MOUSE_ACTIVE_POINTER	//???
 	var/rigged = 0
```
Удали эту переменную, она же нигде не используется у тебя?

Breyn78 committed:
Убран лишний код

убран лишний код добавленный по неопытности, теперь можно залезать на стойки и всё нормально
